p2p/transport/webtransport/transport_test.go

```diff
@@ -133,7 +133,8 @@ func TestTransport(t *testing.T) {
 		require.NoError(t, err)
 		_, port, err := net.SplitHostPort(addr)
 		require.NoError(t, err)
-		require.Equal(t, ma.StringCast(fmt.Sprintf("/ip4/127.0.0.1/udp/%s/quic-v1/webtransport", port)), conn.RemoteMultiaddr())
+		require.Equal(t, ma.StringCast(fmt.Sprintf("/ip4/127.0.0.1/udp/%s/quic-v1/webtransport", port)), stripCertHashes(conn.RemoteMultiaddr()))
+		require.Equal(t, 2, len(extractCertHashes(conn.RemoteMultiaddr())))
```

Comment:
Changes like the ones in this file make me wonder if the fixes here are going to result in anyone's code breaking.